docs/documentation/concepts/units-of-measurement.md

```diff
@@ -71,3 +71,117 @@ When creating QuantityType states the framework offers some useful packages and
 The `org.eclipse.smarthome.core.library.unit` package contains the classes `SIUnits`, `ImperialUnits` and `SmartHomeUnits` which provide units unique to either of the measurement systems and common units used in both systems.
 The `MetricPrefix` class provides prefixes like MILLI, CENTI, HECTO, etc. which are wrappers to create derived units.
 The `org.eclipse.smarthome.core.library.dimension` and `javax.measure.quantity` packages provide interfaces which are used to type the generic QuantityType and units. 
+
+## List of Units
+
+List of the unit available with the UoM (Unit of Measurement) types
+
+
+### Imperial:
+
+
+| Type | Unit | Symbol |
```

Comment:
Could you please format all the tables in a way that they also look nice in markdown? 
You may, of course, use tools like e.g. https://www.tablesgenerator.com/markdown_tables ("File" -> "Paste table data...") or similar and simply copy the resulting markdown in here.